### DISCUSSION TITLE
AI Tech Interview Q/A 형식 PR 문의

### DISCUSSION THREAD

**Discussion by LearningnRunning** [boost-devs/ai-tech-interview#191] [2. 질문]
안녕하세요, AI Tech Interview 레포지토리 관리자님들께 문의드립니다.
저는 현재 여러분의 레포지토리에 있는 md 파일들을 활용하여 모바일 기기에서 쉽게 문제와 정답을 확인할 수 있는 앱을 개발했습니다. 이 앱은 Streamlit을 사용하여 만들어졌으며, 다음 링크에서 확인하실 수 있습니다:
https://ai-tech-interview.streamlit.app/
이 프로젝트가 AI Tech Interview 레포지토리의 목적과 부합하며 많은 사용자들에게 도움이 될 수 있을 것 같아 PR을 고려하고 있습니다. 이와 관련하여 몇 가지 문의사항이 있습니다:

이러한 형태의 앱 버전을 PR로 받아주실 수 있을까요?
만약 가능하다면, 어떤 방식으로 PR을 제출하는 것이 좋을까요? (예: 앱 코드를 별도의 디렉토리에 추가, README에 앱 링크 추가 등)
PR 제출 시 주의해야 할 특별한 사항이 있다면 알려주실 수 있을까요?

PR 작성 규칙을 확인했으며, 다음 사항들을 유념하겠습니다:

"1 문제 1 PR" 원칙 준수
PR 템플릿에 맞춰 작성
PR 제목은 "[분야] 수정 내용" 형식으로 작성
미적 개선 또는 답변 개선에 초점을 맞춘 PR 제출
LaTeX 수식 작성 규칙 준수

이 프로젝트가 AI Tech Interview 레포지토리에 기여할 수 있는 방법에 대해 조언해 주시면 감사하겠습니다. 또한, 이 앱이 레포지토리의 목적과 맞지 않는다고 생각되시면 솔직히 말씀해 주시기 바랍니다.
감사합니다.

**Comment by CoodingPenguin** [comment]
@LearningnRunning 
안녕하세요!

우선 discussion 알림을 켜놓지 않아서 확인이 많이 늦어졌습니다🥲

[공유주신 링크](https://ai-tech-interview.streamlit.app/)를 확인해봤는데 퀴즈/답변 형식으로 면접을 연습하기 좋은 형식으로 되어 있더라구요!
말씀주신 것처럼 이 레포로 면접 연습을 하시는 분이라면 큰 도움이 될 것 같습니다👍👍
이런 좋은 서비스 만들어주셔서 정말 감사드립니다!

하지만 현재 모든 팀원이 취업 후 바빠져서 PR을 통해 간단히 유지보수만하고 있는데요.
개발하신 서비스가 반영 후에 유지보수가 되면 좋지만 그러기 쉽지 않아서 아쉽게도 해당 코드 반영이 어려울 것 같습니다.

다만 다른 분들께 큰 도움이 될 듯하여 [서비스 링크](https://ai-tech-interview.streamlit.app/)를 README에 반영하면 좋을 것 같은데요!
괜찮으시다면 README에 개발한 서비스의 링크를 추가한 PR을 올려주시면 반영하도록 하겠습니다.

한 번 생각해보시고 편하실 때 답변주세요!


**+)**
streamlit 코드도 확인해봤는데 지금은 fork된 레포의 코드를 바라보고 있는 상태로 보이는데요.
upstream 코드를 바라보거나 fork된 코드를 바라보되 주기적으로 upstream 코드로 업데이트해야할 것 같습니다.
전자는 코드 수정이 필요하셔서 번거로우실 수 있을 것 같아 개인적으로는 후자를 추천드리고 싶은데요!
레포가 거의 업데이트 되고 있지 않아서 한 번씩 업데이트해주시면 될 것 같습니다.

**Reply by LearningnRunning** [reply]
@CoodingPenguin 말씀하신 upstream 처리 후에 PR드렸습니다 ㅎㅎㅎ
